packages/synapse-react-client/src/components/GoalsV2/GoalsV2.tsx

```diff
@@ -0,0 +1,226 @@
+          /* TODO - handle case where presigned URL expires */
+        }
+        setAssets(
+          files.requestedFiles
+            .filter(el => el.preSignedURL !== undefined)
+            .map(el => el.preSignedURL!),
+        )
+      } catch (e) {
+        console.error('Error on get data', e)
+        setError(e)
+      }
+    }
+    getData()
+  }, [entityId, accessToken, queryResultBundle])
+
+  const tableIdColumnIndex = getFieldIndex(
+    ExpectedColumns.TABLEID,
+    queryResultBundle,
+  )
+  const countSqlColumnIndex = getFieldIndex(
+    ExpectedColumns.COUNT_SQL,
+    queryResultBundle,
+  )
+
+  const titleColumnIndex = getFieldIndex(
+    ExpectedColumns.TITLE,
+    queryResultBundle,
+  )
+  const summaryColumnIndex = getFieldIndex(
+    ExpectedColumns.SUMMARY,
+    queryResultBundle,
+  )
+  const linkColumnIndex = getFieldIndex(ExpectedColumns.LINK, queryResultBundle)
+
+  const goalsDataArray: GoalsV2DataProps[] =
+    queryResultBundle?.queryResult!.queryResults.rows.map((el, index) => {
+      const values = el.values as string[]
+      if (values.some(value => value === null)) {
+        console.warn('Row has null value(s) when no nulls expected')
+      }
+      const tableId =
+        tableIdColumnIndex > -1 ? values[tableIdColumnIndex] : undefined
+      let countSql
+      if (countSqlColumnIndex > -1 && values[countSqlColumnIndex]) {
+        countSql = values[countSqlColumnIndex]
+      } else if (tableId) {
+        countSql = `SELECT * FROM ${tableId}`
+      }
+      const title = values[titleColumnIndex]
+      const summary = values[summaryColumnIndex]
+      const link = values[linkColumnIndex]
+      const asset = assets?.[index] ?? ''
+      return {
+        countSql,
+        title,
+        summary,
+        link,
+        asset,
+      }
+    }) ?? []
+
+  return (
+    <Box
+      sx={{
+        backgroundColor: 'rgba(0, 128, 0, 0.05)',
+        height: '560px',
+        padding: '80px',
+      }}
+    >
+      <Box
+        sx={{
+          textAlign: 'center',
+          paddingBottom: '30px',
+        }}
+      >
+        <Typography
+          variant="headline1"
+          style={{
+            color: 'grey.1000',
+            fontSize: '32px',
+          }}
+          sx={{
+            pt: '30px',
+            mb: '40px',
+            mx: 'auto',
+            width: 'max-content',
+            borderTop: '3px solid rgba(128, 128, 128, 0.25)',
+          }}
+        >
+          What's in the Portal?
+        </Typography>
+        <Button
+          href={dataLink}
+          target="_blank"
+          rel="noopener noreferrer"
+          variant="contained"
+          sx={{
+            backgroundColor: '#5BA998',
+            border: '1px solid white',
+            boxShadow: 'none !important',
+            borderRadius: '5px',
+            '&:hover': {
+              backgroundColor: darken('#5BA998', 0.05),
+              boxShadow: 'none !important',
+            },
+          }}
```

Comment:
Putting ELITE portal theme colors here will make this component non-generic. I think many of the required colors have already been added to the ELITE portal homepage theme (HomePageThemeProvider.tsx), so I think you can probably remove most/all of these inline styles. Any styles that should be applied to all buttons on the ELITE portal home page can be added to that theme file.